test/e2e-tests.sh

```diff
@@ -25,13 +25,148 @@
 
 source $(dirname $0)/../scripts/e2e-tests.sh
 
+# Read metadata.json and get value for key
+# Parameters: $1 - Key for metadata
+function get_meta_value() {
+  # local metadata="${ARTIFACTS}/metadata.json"
+  # cat ${metadata} | grep -oEi "\"${1}\" ?: ?\".*\"" | cut -d "\"" -f 4
+  go run knative.dev/test-infra/test/metahelper --get $@
+}
+
 function knative_setup() {
   start_latest_knative_serving
 }
 
+# Run a prow-cluster-operation tool, installing it first if necessary.
+#             $1..$n - parameters passed to the tool.
+function run_prow_cluster_tool() {
+  go run knative.dev/test-infra/test/prow-cluster-operation $@
```

Comment:
Either use `run_go_tool` to make the "installing it first" part true, of remove it from the comment.